File: back-end/user/src/main/java/com/sensormanagement/user/Models/loginModel.java
package com.sensormanagement.user.Models;

public class loginModel {
    String userId;
    String usrRoleId;
    String orgId;

    public loginModel() {
    }

    public loginModel(String userId, String usrRoleId, String orgId) {
        this.userId = userId;
        this.usrRoleId = usrRoleId;
        this.orgId = orgId;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getUsrRoleId() {
        return usrRoleId;
    }

    public void setUsrRoleId(String usrRoleId) {
        this.usrRoleId = usrRoleId;
    }

    public String getOrgId() {
        return orgId;
    }

    public void setOrgId(String orgId) {
        this.orgId = orgId;
    }
}

File: front-end/src/app/components/Models/loginModel.ts
export class loginModel{
    userId:string;
    usrRoleId:string;
    orgId:string;
    constructor( userId:string, usrRoleId:string, orgId:string)
    {
        this.userId = userId;
        this.usrRoleId = usrRoleId;
        this.orgId = orgId;
    }
}